client/signup/steps/domains/index.jsx

```diff
@@ -655,7 +656,19 @@ export class RenderDomainsStep extends Component {
 			const sortedProducts = this.sortProductsByPriceDescending( productsInCart );
 
 			// Replace the products in the cart with the freshly sorted products.
-			await this.props.shoppingCartManager.replaceProductsInCart( sortedProducts );
+			await this.props.shoppingCartManager
+				.replaceProductsInCart( sortedProducts )
+				.then( () => {
+					this.setState( { replaceDomainFailedMessage: null } );
+				} )
+				.catch( () => {
+					this.setState( {
+						replaceDomainFailedMessage: this.props.translate(
+							'Sorry, there was an error adding that domain. Please refresh the page to try again.'
```

Comment:
As the buttons are unfrozen now, perhaps:
```suggestion
							'Sorry, there was an error adding that domain. Please try again later.'
```

We're not sure if refreshing the page would help with their issue.
What do you think?